crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -253,7 +253,7 @@ impl Entity {
     /// # use bevy_ecs::{prelude::*, component::*};
     /// # use bevy_reflect::Reflect;
     /// #[derive(Reflect, Component)]
-    /// #[reflect(Component)]
+    /// #[reflect(ComponentMut, Component)]
```

Comment:
I noticed that this one has both, but the others (that I've read so far) were just changed to ComponentMut. Is that intentional?

Reply:
It's not intentional, all cases of `#[reflect(Component)]` should be updated to `#[reflect(Component, ComponentMut)]` to preserve previous behaviour. I've made a few changes to fix some of the inconsistencies.